apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommandTextBuilder.cs

```diff
@@ -37,14 +39,18 @@ public sealed class SpannerCommandTextBuilder
         private string _targetTable;
 
         /// <summary>
+        /// Gets the resulting string to be used for <see cref="SpannerCommand.CommandText"/>.
         /// </summary>
         public string CommandText { get; private set; }
 
         /// <summary>
+        /// Gets the type of Spanner command (Select, Update, Delete, InsertOrUpdate, Insert, Ddl)
```

Comment:
Period here and below (only pointing out as you mentioned you'd done another sweep already).